ui/externs/ui.js

```diff
@@ -247,6 +248,9 @@ shaka.extern.UIVolumeBarColors;
  *   Indicate the default VR projection mode.
  *   Possible values: <code>'equirectangular'</code> or <code>'cubemap'</code>.
  *   Defaults to <code>'equirectangular'</code>.
+ * @property {boolean} setupMediaSession
+ *   If true, MediaSession controls will managed by the UI.
```

Comment:
It's not obvious from this description that setting this to `true` will also cause the UI to use `APIC` metadata to change the video poster. That should probably be mentioned too.

Reply:
Updated the description